model/proxy_service_notify.go

```diff
@@ -0,0 +1,18 @@
+package model
+
```

Comment:
Change the name of this file to something more explicit since it is in the `model` package. It can be something like `perf_proxy_service_notify.go`.